### ISSUE TITLE
pip-installed wheel generates travis spam on macOS

### ISSUE THREAD

**Issue by fluffy-critter** [ionelmc/python-lazy-object-proxy#29]
Hi, on some other Python projects (notably pylint), this library is generating spam on standard error at time of exit. For example:

```
$ pipenv run pylint -f colorized publ

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

profiling: /Users/travis/build/ionelmc/python-lazy-object-proxy/build/temp.macosx-10.13-x86_64-3.7/src/lazy_object_proxy/cext.gcda: cannot open: No such file or directory
```

Related pylint issue: https://github.com/PyCQA/pylint/issues/2907

This happens on macOS 10.14 using lazy-object-proxy v1.4.0; pinning my environment's version to 1.3.1 fixes the issue (workaround found by @wizzl35, see https://github.com/PyCQA/pylint/issues/2907#issuecomment-491001066)

**Comment by allwu** [comment]
I saw the same issue when using `pylint` in macOS 14.14.4

**Comment by ionelmc** [comment]
Oooof ... looks like I build wheels with coverage cflags. I'll fix this.

**Comment by ionelmc** [comment]
New wheels being built at https://test.pypi.org/project/lazy-object-proxy/1.4.1.dev3/#files - try and let me know if osx problem is gone.

**Comment by fluffy-critter** [comment]
Yes, that seems to fix the problem on macOS.

**Comment by ionelmc** [comment]
Published v1.4.1
